simulation/CMakeLists.txt

```diff
@@ -38,8 +42,9 @@ include_directories("${CMAKE_CURRENT_SOURCE_DIR}/include")
 
 PCL_ADD_LIBRARY(${LIB_NAME} COMPONENT ${SUBSYS_NAME} SOURCES ${srcs} ${incs})
 
+set(_glew_target $<IF:$<BOOL:PCL_SHARED_LIB>,GLEW::glew,GLEW::glew_s>)
```

Comment:
`GLEW::GLEW` is not properly defined, so I explicitly link the either the shared or static lib depending on how the rest of pcl's libs are built.

Reply:
Missing S here as well in PCL_SHARED_LIB(S)